lang/javascript/javascript.py

```diff
@@ -44,6 +44,32 @@
     "then": "then",
 }
 
+ctx.lists["user.code_keyword"] = {
```

Comment:
Note that some of the keywords have trailing space, depending how they're usually used